Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1706,35 +1708,73 @@ public extension Tensor where Scalar: Numeric & Comparable {
 }
 
 internal extension Tensor where Scalar: TensorFlowFloatingPoint {
-  @inlinable
-  func _vjpMinOrMax(squeezingAxes axes: Tensor<Int32>) -> (Tensor, (Tensor) -> Tensor) {
-    let result = max(squeezingAxes: axes)
```

Comment:
Note: here's the issue in the old implementation.

`_vjpMinOrMax(squeezingAxes:)` is registered as the derivative of `min(squeezingAxes:)`, but it returns `max(squeezingAxes: axes)` as the original result.